README.md

```diff
@@ -513,6 +515,26 @@ To add RAC to your application:
 To see a project already set up with RAC, check out [C-41][] or [GroceryList][],
 which are real iOS apps written using ReactiveObjC.
 
+#### Carthage
```

Comment:
Should this go above the submodule section (the same way it is in ReactiveCocoa)? It seems like Carthage or CocoaPods would be the most common installation routes over submodules.